internal/provider/organization_role_membership_resource_test.go

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package provider
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+func TestAccOrganizationRoleMembershipResource(t *testing.T) {
```

Comment:
@iverberk - for testing in parallel - please can you generate dynamic test data as per example [here](https://github.com/sonatype-nexus-community/terraform-provider-sonatypeiq/blob/a75d10a8662a94cb19c50c16fc3cb8780be774f7/internal/provider/organization_resource_test.go#L61) ?